Nodejs/Product/TestAdapter/TestFrameworks/Jest/jest.js

```diff
@@ -116,5 +123,36 @@ function logError() {
     console.error.apply(console, errorArgs);
 }
 
+function readConfigs(projectFolder, context)
+{
+    // First look for Jest.config.js, otherwise look at the package.json under "jest" tag
+    var userConfig;
+    const jestConfigPath = projectFolder + "\\jest.config.js";
+    const packageJsonPath = projectFolder + "\\package.json";
+
+    if(fs.existsSync(jestConfigPath))
+    {
+        userConfig = require(jestConfigPath);
+        mergeConfigs();
+        return userConfig;
+    }
+
+    if(fs.existsSync(packageJsonPath))
+    {
+        userConfig = require(packageJsonPath).jest;
+        mergeConfigs();
+        return userConfig;
+    }
+
+    function mergeConfigs()
+    {
+        // If no config was found OR the user doesn't have these tags set up, add it.
+        if(!userConfig) userConfig = {};
+        if(!userConfig.setupFilesAfterEnv) userConfig.setupFilesAfterEnv = ["<rootDir>/src/setupTests.js"];
```

Comment:
Seems like this configurations are the defaults used by React rather than the ones used by Jest. 

I'm not sure if this is the correct approach for NTVS as assuming a user is creating a plain JS project and adding jest support, this configurations would be incorrect for him.